services/web/server/src/simcore_service_webserver/rest.py

```diff
@@ -58,20 +63,13 @@ def setup_rest(app: web.Application):
             f"__version__.api_version_prefix {api_version_prefix} does not fit openapi.yml version {specs.info.version}"
         )
 
+    # defines healthcheck mechanism
+    healthcheck = HeathCheck()
+    healthcheck.on_healthcheck.append(_log_check)
```

Comment:
Not sure I understand why call a function to log in the healtcheck. I would only print something to the logs if there was an issue with it.

Reply:
removed. was a dummy test that i forgot.